### PR TITLE
Translated @FindFile help page to English in FarEng.hlf.m4.

### PR DESCRIPTION
**Pull request by MKadaner**
Also, consistently replaced "code page" with "codepage" (without space) in the help and resource files. The latter spelling seems to be prevalent in the existing texts.

### CODE REVIEW

File: far/FarEng.hlf.m4 [hunk 1]
```diff
@@ -1688,19 +1688,19 @@ separators, which by default are: #!%^&*()+|{}:"<>?`-=\\[];',./#.
...
-кодовых страниц можно настроить в ~меню выбора кодовой страницы~@CodepagesMenu@
-настроек, редактора или программы просмотра). При необходимости поиска по собственному
-набору кодовых страниц отметьте их в списке при помощи клавиш #Ins# или #Space# и выберите
-пункт #Отмеченные кодовые страницы#.
+    Dropdown list #Using code page# allows to choose a specific codepage
+to be used for the search. If the item #All standard codepages# is selected
+in the dropdown list, Far will use all standard codepages (ANSI, OEM, UTF-8,
+UTF-16, UTF-16 BE), as well as #Favorite# codepages (the list of #favorite#
+codepages can be specified in the ~Codepages menu~@CodepagesMenu@
+in the options, editor, or viewer). To search using custom set of codepages,
```
[the remark this answers is not in the document]
Comment (w17) [reply far/FarEng.hlf.m4]: editor, viewer: Shift+F8 F1

**Comment by alabuzhev** [comment]
I'm not so sure about replacing "code page" with "codepage".
[MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/dd317756.aspx) and [Wiki](https://en.wikipedia.org/wiki/Code_page) use "code page" spelling and Google suggests it too.
Correctness is probably more important here than the current prevalence.

File: far/FarEng.hlf.m4 (hunk 1 above)
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: Shouldn't it be "using **a** custom set"?

[the remark this answers is not in the document]
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Got it! Thank you.

Comment (MKadaner) [reply far/FarEng.hlf.m4]: Of course it should! Will fix.

**Comment by MKadaner** [comment]
I think you are right about code pages. I did some research yesterday, not just scanned Far resources, but obviously not thorough enough. I will change everything to read "code page".

**Comment by alabuzhev** [comment]
Looks good. Could you squash these two commits into one please?

File: far/changelog_eng [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+
+zg 01.02.2018 22:12:58 +0200 - build 5135
+
+1. In the part-screen editor/viewer, only editor/viewer was redrawn after CtrlO.
+
+2. In the part-screen editor/viewer, CtrlB/CtrlShiftB did not work.
+
+3. Forum: In the part-screen editor/viewer, hotkey panel sometimes did not refresh.
+
+MZK 28.01.2018 15:50:33 -0800
```
Comment (alabuzhev) [inline far/changelog_eng]: Lines 19-21 are already here.

Comment (MKadaner) [reply far/changelog_eng]: Already fixed. Sorry for the mess; I am new to git.

**Comment by alabuzhev** [comment]
Thanks!

**Comment by MKadaner** [comment]
Squashed and rebased. The change looks as intended.

**Comment by MKadaner** [comment]
Thank you!

**Comment by VictorVG** [comment]
Не исправлено - https://github.com/FarGroup/FarManager/issues/19 !

**Comment by VictorVG** [comment]
Не исправлено - https://github.com/FarGroup/FarManager/issues/19:

> Опечатка в far/FarEng.hlf.m4::5248  - Version invformation :)

и на форумах  в неё пальцем тычут. Делать так сразу на совесть. Чего ляпы оставлять? На развод половецкой стране?

**Pull request merged**